tests/tests_integration/test_file_format_integration/test_excel_integration.py

```diff
@@ -30,7 +30,7 @@ def test_excel_reader_with_infer_schema(
 ):
     """Reading CSV files with inferSchema=True working as expected on any Spark, Python and Java versions"""
     spark_version = get_spark_version(spark)
-    if spark_version < (3, 2):
+    if spark_version < "3.2":
```

Comment:
```suggestion
    if spark_version <  Version("3.2"):
```
И ниже тоже проверь, по вхождению `< "`